pyramid_oereb/core/config.py

```diff
@@ -886,7 +947,10 @@ def _read_map_layering():
             map_layering_config.get('source').get('class'),
             **map_layering_config.get('source').get('params'))
 
-        return map_layering_reader.read()
+        map_layering_record = map_layering_reader.read()
+        if len(map_layering_record) == 0:
+            log.error('No records found for map layering.')
```

Comment:
I think it is not necessarily an error if the table is empty.

Reply:
Thanks for the review. I think you're right. How about logging a warning? It could still be unintentional to have the table empty? Same for availability.

Reply:
A warning seems good to me. Or even just an info.